app/src/main/java/com/github/fribourgsdp/radio/AddPlaylistActivity.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.fribourgsdp.radio
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+
+class AddPlaylistActivity : AppCompatActivity() {
+    private val listSongs = ArrayList<Song>()
+    private var listNames = ArrayList<String>()
+    lateinit var listAdapter: ArrayAdapter<String>
+    lateinit var listView : ListView
+    lateinit var errorTextView : TextView
+    lateinit var user : User
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_add_playlist)
+        listAdapter = ArrayAdapter(this, android.R.layout.simple_list_item_1, listNames)
+        listView = findViewById(android.R.id.list)
+        listView.adapter = listAdapter
+        errorTextView = findViewById(R.id.addPlaylistErrorTextView)
+
+        user = UserProfileActivity.loadUser(this)
+    }
+    fun addItems(view : View) : Unit {
```

Comment:
Do you plan to add tests to check that users are entering songs that actually exist?

Reply:
No, since if they don't write songs that exist, they will be able to enter their own lyrics (if musixmatch does not find them). That's Alex' task for this week. Adding song verification is another story.